README.md

```diff
@@ -364,6 +364,38 @@ You can now submit a new commit/change that will update the CI system to run a n
 
 It might happen that the CI does not get reloaded so you can simply leave a comment on your PR with "test this please" and it will trigger a new CI build.
 
+# Backporting changes
+
+If a change should be backported to a `stable-*` Git branch:
+
+* Mark your PR with the GitHub label "Backport" so we don't lose track of it.
+* Determine the latest available stable branch:
+  `git branch -r | grep origin/stable`
```

Comment:
What about this?
git branch -r --list "origin/stable-[0-9].[0-9]" | sort -r | sed 1q